packages/epics/src/organisation/components/space-group-slider.tsx

```diff
@@ -0,0 +1,69 @@
+'use client';
+
+import { Text } from '@radix-ui/themes';
+import {
+  Carousel,
+  CarouselContent,
+  CarouselItem,
+  Button,
+} from '@hypha-platform/ui';
+import Link from 'next/link';
+import { SpaceType } from '@hypha-platform/graphql/rsc';
+import { Locale } from '@hypha-platform/i18n';
+import { CardOrganisation } from './card-organisation';
+
+type SpaceGroupSliderProps = {
+  spaces?: SpaceType[];
+  isLoading?: boolean;
+  lang?: Locale;
+  type?: string;
+};
+
+export const SpaceGroupSlider = ({
+  spaces,
+  isLoading,
+  lang,
+  type,
+}: SpaceGroupSliderProps) => {
+  const getDhoPathAgreements = (lang: Locale, id: string) => {
+    return `/${lang}/dho/${id}/agreements`;
+  };
```

Comment:
Sorry, but this is still not correct. The component should not care about the router we use. We could decide tomorrow to use remix, then these things fall apart (just an example). Like already discussed, either pass down the compiled `href`, or pass in the getter function which only requres the `id`. Like:

```
type SpaceGroupSliderProps = {
  spaces?: SpaceType[];
  isLoading?: boolean;
  type?: string;
  getHref: (id: string) => string
};
```